docs/guides/contribute/createnew.md

```diff
@@ -9,40 +9,59 @@ tags:
 
 # How To Create a New Document in GitHub
 
-_When you are ready to submit your original written documentation for approval, follow these easy steps:_
-
+*When you are ready to submit your original written documentation for approval, follow these easy steps:*
 
 ## With the GitHub GUI
 
 You can complete almost all tasks from the web GUI on GitHub. Here's an example of adding a file you've created on your local machine to the Rocky Linux documentation GitHub repository.
 
-
-
 1. Login to your GitHub account.
-2. Navigate to the Rocky Linux Documentation repository at [https://github.com/rocky-linux/documentation](https://github.com/rocky-linux/documentation).
+2. Navigate to the Rocky Linux Documentation repository at <https://github.com/rocky-linux/documentation>.
 3. You should be on the "Main" branch, so check the drop-down label down in the middle section to be sure you are. Your document might not ultimately end up in the "Main" branch, but admins will move it around to where it logically fits later.
-4. On the right-hand side of the page, click the "Fork" button, which will create your copy of the documentation.
-5. In the middle of the page on the forked copy, just to the left of the Green "Code" drop-down, is an "Add file" button. Click this and choose the "Upload files" option.
+4. On the right-hand side of the page, click the ++"Fork"++ button, which will create your copy of the documentation.
+5. In the middle of the page on the forked copy, just to the left of the Green "Code" drop-down, is an ++"Add file"++ button. Click this and choose the "Upload files" option.
 6. This will allow you to drag and drop files here or browse to them on your computer. Go ahead and use the method that you prefer.
 7. Once the file is uploaded, the next thing you need to do is create a Pull Request. This request lets the upstream administrators know you have a new file (or files) that you want them to merge with the master branch.
-8. Click on "Pull Request" in the upper-left corner of the screen.
+8. Click on `Pull Request` in the upper-left corner of the screen.
```

Comment:
Technically @nishaaaaaant , "Pull Request" is not a code block. Is there a better way to format it w/o using the backticks or quotes? Otherwise, I don't see anything not worthy of merging. Take a look and perhaps fix that, or provide some justification for it if you feel strongly about it. Thank you!

Reply:
I can change it to 
Click on the ++"Pull Request"++ button in the upper-left corner of the screen.
Since it is a button.

Reply:
Note worthy merging for the whole PR or just the createnew.md file?